ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutor.java

```diff
@@ -219,7 +219,7 @@ private static void validateTopicData(
       final boolean ranWithInsertStatements
   ) {
     if (actual.size() != expected.size()) {
-      throw new KsqlException("Expected <" + expected.size()
+      throw new KsqlException("Topic " + topicName + "Expected <" + expected.size()
```

Comment:
Seems you need a space after `topicName + " Expected"`?

Reply:
Good catch. Thanks.